incubator/hnc/e2e/tests/delete_crd_test.go

```diff
@@ -0,0 +1,55 @@
+package test
+
+import (
+	"os/exec"
+	"os"
+	"time"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Delete-anchor-crd", func() {
+	
+	AfterEach(func() {
+		// clean up
+		nses := []string{
+			"delete-crd-parent",
+			"delete-crd-child",
+		}
+
+		for _, ns := range nses {
+			runSuccess("kubectl hns set", ns, "-a")
+			//remove subname annotation
```

Comment:
Comment: "Remove all possible objections HNC might have to deleting a network. Make sure it has cascading deletion so we can delete any of its subnamespace descendants, and make sure that it's not a subnamespace itself so we can delete it directly."

Reply:
done